app/templates/admin/view_tags.html

```diff
@@ -47,6 +48,7 @@ <h2 class="ui header">
                                 <tr>
                                     <td>{{ t.tag_name }}</td>
                                     <td class="user role">{{t.tag_type.tag_type_name}}</td>
+                                    <td><button onclick="window.location.href = '{{ url_for('admin.delete_tag', tag_id=t.id) }}';" class="btn btn-primary delete-tag"> Delete Tag </button> <button onclick="window.location.href = '{{ url_for('admin.edit_tag', tag_id=t.id) }}';" class="btn btn-primary delete-tag"> Edit Tag </button></td>

```

Comment:
can u change the color of the delete button?

Reply:
done